config/default.json

```diff
@@ -1,7 +1,5 @@
 {
-  "smtp": {
-    "host": "127.0.0.1",
-    "port": 25,
-    "sender": "Firefox Accounts <accounts@firefox.com>"
-  }
+  "sender": "Firefox Accounts <accounts@firefox.com>",
```

Comment:
Oh and, other thing I forgot to mention, `sender` moved up to the top level semantically because it's not specific to the `smtp` settings (it's also used in the SES branch, which I'm hoping to PR tomorrow).